app/src/device-info.cpp

```diff
@@ -24,7 +24,7 @@ DeviceInfo::DeviceInfo()
 //------------------------------------------------------------------------------
 String DeviceInfo::latest()
 {
-    String res = ">>>>>>>>>>>>>>>>>>>> DeviceInfo <<<<<<<<<<<<<<<<<<<<\r\n";
+    String res = "\033[1;35m >>>>>>>>>>>>>>>>>>>> DeviceInfo <<<<<<<<<<<<<<<<<<<< \033[1;39m \r\n";

```

Comment:
Repeated code in many places. Maybe create a proper macro for this functionality?

Reply:
marco done